### PR TITLE
New Behringer-CMD-PL-1 controller

### PR DESCRIPTION
**Pull request by yledoare**

### CODE REVIEW

File: res/controllers/Behringer CMD PL-1.midi.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <author>Yann Le Doaré</author>
+        <description>Behringer CMD PL-1 / Mixxx controller mapping </description>
+        <forums>http://www.mixxx.org/forums/yann</forums>
+    </info>
+    <controller id="CMD">
+        <scriptfiles>
+            <file functionprefix="CMDPL1" filename="Behringer-CMD-PL-1-scripts.js"/>
+        </scriptfiles>
+        <controls>
+            <control> <group>[Channel1]</group> <key>play</key> <description>Play</description> <status>0x90</status> <midino>0x23</midino> <options> <switch/> </options> </control>
```
Comment (Holzhaus) [inline res/controllers/Behringer CMD PL-1.midi.xml:14]: Please split these lines and indent them properly, like this:

```suggestion
            <control>
                <group>[Channel1]</group>
                <key>play</key>
                <description>Play</description>
                <status>0x90</status>
                <midino>0x23</midino>
                <options>
                    <switch/>
                </options>
            </control>
```

**Review by Holzhaus — CHANGES_REQUESTED** [review]
Thank you for contributing!

Would you please [sign our contributor agreement form](https://docs.google.com/forms/d/e/1FAIpQLScC9QG327sjLL0eWftmfGUasxFFLxg6LCXJ2xHDYRzFIRqyiw/viewform?formkey=dEpYN2NkVEFnWWQzbkFfM0ZYYUZ5X2c6MQ) for this and all future contributions?

Please add a wiki page for this controller and document the mapping there. That will allow us to review this properly. Also, please rebase onto the 2.2 or 2.3 branch and change the target branch of this PR to that branch: https://mixxx.org/wiki/doku.php/using_git#targeting_another_base_branch

**Comment by Holzhaus** [comment]
Ping @yledoare.

**Comment by yledoare** [comment]
> Ping @yledoare.

Hello @Holzhaus , I will try to do it this week !

**Comment by yledoare** [comment]
Done : https://github.com/mixxxdj/mixxx/pull/3147

**Pull request closed without merging**